draft-ietf-tls-dnssec-chain-extension-08.xml

```diff
@@ -376,23 +396,42 @@
       </t>
 
       <t>
-        The first RRset in the chain MUST contain the TLSA record set
-        being presented. However, if the owner name of the TLSA record
-        set is an alias (CNAME or DNAME), then it MUST be preceded by the
-        chain of alias records needed to resolve it. DNAME chains SHOULD
-        omit unsigned CNAME records that may have been synthesized in the
-        response from a DNS resolver. (If unsigned synthetic CNAMES are
-        present, then the TLS client will just ignore them, as they are
-        not necessary to validate the chain.)
+        The chain of records appears in logical order for validation.
+        If the owner name of the TLSA RRset is an alias (e.g., CNAME or DNAME),
+        then the chain of signed alias records MUST appear first in the
+        AuthenticationChain; otherwise, the TLSA RRset appears first.
+        If at some point along the alias chain the
+        target of an alias leads to an unsigned zone, the response will include a
+        denial of existence response proving insecure delegation of that
+        zone.  DNAME chains SHOULD omit unsigned CNAME records that may have
```

Comment:
(Assuming I understand the question correctly) If the client doing DANE opportunistically (when TLSA records are present), then denial of existence would return the client to the status quo ante, where it does WebPKI or whatever it would have done otherwise. If DANE is mandatory, then the client will abort the connection.